example/gardener-local/gardenlet/values-kind2.yaml

```diff
@@ -0,0 +1,30 @@
+global:
+  gardenlet:
+    config:
+      gardenClientConnection:
+        bootstrapKubeconfig:
+          kubeconfig: |
+            apiVersion: v1
+            kind: Config
+            current-context: gardenlet-bootstrap
+            clusters:
+            - cluster:
+                insecure-skip-tls-verify: true
+                server: https://gardener-local-control-plane:6443
+              name: default
+            contexts:
+            - context:
+                cluster: default
+                user: gardenlet-bootstrap
+              name: gardenlet-bootstrap
+            users:
+            - name: gardenlet-bootstrap
+              user:
+                token: 07401c.f395accd246ae52d
+      seedConfig:
+        metadata:
+          name: local2
+        spec:
+          ingress:
+            domain: ingress.local2.seed.local.gardener.cloud
+          secretRef: {}
```

Comment:
This is something I wondered how to do with minimal changes. 
`spec.secretRef` is not required when the seedConfig is part of the `gardenlet` configuration and the `gardenlet` uses a bootstrap kubeconfig (I could potentially also remove it from the `example/gardener-local/gardenlet/values.yaml` file). And currently there is no problem to reuse the `seed-local` backup secret also for the second seed cluster because it doesn't contain any backup-related credentials anyway.
This is because when using skaffold the `seed-local2` secret is not deployed since it is part of the `example/provider-local/seed-kind2` kustomize directory. 

As an alternative we could introduce a completely separate dummy secret that is used only for backups and specify it in all seed configs.

Reply:
https://github.com/gardener/gardener/blob/6af87c88f796d53d23776a8839b63145cf108808/example/provider-local/base/kustomization.yaml#L7-L11 should probably be moved to `example/provider-local/seed-kind/kustomization.yaml` now, right? You are already doing the same in https://github.com/gardener/gardener/pull/6059/files#diff-1aa685072589ad170f9bb94e667d1b05b1b7df598e2cc19df490ead44d03da19R10-R14 and it would be symmetric then. In this case you could patch the names also in `values-kind2.yaml`?